homeassistant/components/samsungtv/bridge.py

```diff
@@ -90,14 +90,6 @@ async def async_get_device_info(
         if info := await bridge.async_device_info():
             return port, METHOD_WEBSOCKET, info
 
-    # Try encrypted websocket port
-    bridge = SamsungTVBridge.get_bridge(
-        hass, METHOD_ENCRYPTED_WEBSOCKET, host, ENCRYPTED_WEBSOCKET_PORT
-    )
-    result = await bridge.async_try_connect()
-    if result == RESULT_SUCCESS:
-        return port, METHOD_ENCRYPTED_WEBSOCKET, await bridge.async_device_info()
-
```

Comment:
I think this check need to come before the non-encrypted check and see if this url works https://github.com/xchwarze/samsung-tv-ws-api/blob/master/samsungtvws/encrypted/remote.py#L102